requirements.txt

```diff
@@ -46,7 +46,7 @@ requests==2.22.0
 logilab-common==1.4.3
 logilab-astng==0.24.3
 editdistance==0.5.3
-pylint==1.9.5 # pyup: <2.0.0
+pylint==1.9.4 # pyup: <2.0.0
```

Comment:
was this downgraded accidentally? otherwise looks good - i'll merge.

Reply:
Huh, actually that was intentional because venv couldn't find a satisfying dependency, but I'm not sure why.